dbos/admin_server.go

```diff
@@ -378,27 +378,18 @@ func newAdminServer(ctx *dbosContext, port int) *adminServer {
 			}
 		}
 
-		req.Status = "" // We are not expecting a filter here but clear just in case
```

Comment:
Because the console never passes a status, I previously just dismissed any status filter received on the queues endpoint. I think this is wrong, so this PR only sets enqueued/pending if no status is present in the request.